offchain-modules/bridge-cli/src/commands/types.rs

```diff
@@ -0,0 +1,91 @@
+use clap::Clap;
+
+#[derive(Clap, Clone, Debug)]
+#[clap(version = "0.1", author = "LeonLi000 <matrix.skygirl@gmail.com>")]
+pub struct Opts {
+    #[clap(subcommand)]
+    pub subcmd: SubCommand,
+}
+
+#[derive(Clap, Clone, Debug)]
+pub enum SubCommand {
+    TransferToCkb(TransferToCkbArgs),
+    Approve(ApproveArgs),
+    Lock(LockArgs),
+    ParseEthProof(ParseEthProofArgs),
+    VerifyEthSpvProof(VerifyEthSpvProofArgs),
```

Comment:
What is this command for?

Reply:
This command is to submit eth spv proof to ckb chain and verify it. If the verification is passed, it will issue new token

Reply:
Then `mint` may be a better name. It is our aim, verify the proof is just one of the steps, there are other steps like parse and verify the event.

Reply:
nod.